pkg/cloud-provider/cloudapi/internal/cloud_common.go

```diff
@@ -230,14 +224,14 @@ func (c *cloudCommon) GetStatus(accountNamespacedName *types.NamespacedName) (*c
 	return accCfg.GetStatus(), nil
 }
 
-// AddInventoryPoller adds a periodic poller for fetching cloud inventory.
-func (c *cloudCommon) AddInventoryPoller(accountNamespacedName *types.NamespacedName) error {
+// DoInventoryPoll adds a periodic poller for fetching cloud inventory.
```

Comment:
is the comment correct? are you adding inventory poller here?

Reply:
Updated it.